queue_classic_admin.gemspec

```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib,vendor}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency "rails", ">= 4.0.9"
-  s.add_dependency "queue_classic", "~> 3.0"
+  s.add_dependency "rails", ">= 4.0.9", "< 5.1"
```

Comment:
What breaks on 5.1 ?

Reply:
Mostly tests because of the format - I'll look at making sure it works later this week

Reply:
👀